[2024-12-18T11:54:44.539+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-18T11:54:44.564+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: snowflake_connection_check.check_snowflake_connection manual__2024-12-18T11:49:38.971052+00:00 [queued]>
[2024-12-18T11:54:44.576+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: snowflake_connection_check.check_snowflake_connection manual__2024-12-18T11:49:38.971052+00:00 [queued]>
[2024-12-18T11:54:44.578+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2024-12-18T11:54:44.598+0000] {taskinstance.py:2889} INFO - Executing <Task(SnowflakeOperator): check_snowflake_connection> on 2024-12-18 11:49:38.971052+00:00
[2024-12-18T11:54:44.608+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1785) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-18T11:54:44.610+0000] {standard_task_runner.py:72} INFO - Started process 1787 to run task
[2024-12-18T11:54:44.611+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'snowflake_connection_check', 'check_snowflake_connection', 'manual__2024-12-18T11:49:38.971052+00:00', '--job-id', '2578', '--raw', '--subdir', 'DAGS_FOLDER/snowflake_connector_dag.py', '--cfg-path', '/tmp/tmptu5dl106']
[2024-12-18T11:54:44.614+0000] {standard_task_runner.py:105} INFO - Job 2578: Subtask check_snowflake_connection
[2024-12-18T11:54:44.676+0000] {task_command.py:467} INFO - Running <TaskInstance: snowflake_connection_check.check_snowflake_connection manual__2024-12-18T11:49:38.971052+00:00 [running]> on host 8348eb091483
[2024-12-18T11:54:44.791+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='snowflake_connection_check' AIRFLOW_CTX_TASK_ID='check_snowflake_connection' AIRFLOW_CTX_EXECUTION_DATE='2024-12-18T11:49:38.971052+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-18T11:49:38.971052+00:00'
[2024-12-18T11:54:44.792+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-18T11:54:44.808+0000] {sql.py:278} INFO - Executing:  select current_warehouse();
[2024-12-18T11:54:44.823+0000] {base.py:84} INFO - Retrieving connection 'snowflake_con'
[2024-12-18T11:54:44.836+0000] {base.py:84} INFO - Retrieving connection 'snowflake_con'
[2024-12-18T11:54:44.838+0000] {connection.py:414} INFO - Snowflake Connector for Python Version: 3.12.4, Python Version: 3.12.8, Platform: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-12-18T11:54:44.840+0000] {connection.py:1197} INFO - Connecting to GLOBAL Snowflake domain
[2024-12-18T11:54:44.841+0000] {connection.py:1278} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-12-18T11:54:45.364+0000] {cursor.py:1166} INFO - Number of results in first chunk: 1
[2024-12-18T11:54:45.365+0000] {sql.py:553} INFO - Running statement: select current_warehouse();, parameters: None
[2024-12-18T11:54:45.415+0000] {cursor.py:1166} INFO - Number of results in first chunk: 1
[2024-12-18T11:54:45.416+0000] {sql.py:562} INFO - Rows affected: 1
[2024-12-18T11:54:45.418+0000] {snowflake.py:447} INFO - Rows affected: 1
[2024-12-18T11:54:45.420+0000] {snowflake.py:448} INFO - Snowflake query id: 01b91caa-0000-d1e7-0008-6fb60004ddce
[2024-12-18T11:54:45.480+0000] {cursor.py:1166} INFO - Number of results in first chunk: 1
[2024-12-18T11:54:45.481+0000] {connection.py:789} INFO - closed
[2024-12-18T11:54:45.515+0000] {connection.py:795} INFO - No async queries seem to be running, deleting session
[2024-12-18T11:54:45.603+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-18T11:54:45.605+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=snowflake_connection_check, task_id=check_snowflake_connection, run_id=manual__2024-12-18T11:49:38.971052+00:00, execution_date=20241218T114938, start_date=20241218T115444, end_date=20241218T115445
[2024-12-18T11:54:45.672+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-18T11:54:45.702+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-18T11:54:45.707+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
